src/cli/commands/force/lightning/local/device/create.ts

```diff
@@ -150,27 +150,30 @@ export class Create extends Setup {
     }
 
     private async executeAndroidDeviceCreate(): Promise<any> {
-        const preferredPack = await AndroidSDKUtils.findRequiredEmulatorImages();
-        const emuImage = preferredPack.platformEmulatorImage || 'default';
-        const androidApi = preferredPack.platformAPI;
-        const abi = preferredPack.abi;
-
-        return AndroidSDKUtils.createNewVirtualDevice(
-            this.deviceName,
-            emuImage,
-            androidApi,
-            this.deviceType,
-            abi
+        return AndroidSDKUtils.findRequiredEmulatorImages().then(
+            (preferredPack) => {
+                const emuImage =
+                    preferredPack.platformEmulatorImage || 'default';
+                const androidApi = preferredPack.platformAPI;
+                const abi = preferredPack.abi;
+                return AndroidSDKUtils.createNewVirtualDevice(
+                    this.deviceName,
+                    emuImage,
+                    androidApi,
+                    this.deviceType,
+                    abi
+                );
+            }
         );
     }
 
     private async executeIOSDeviceCreate(): Promise<any> {
```

Comment:
Same question on return type here, except this would be `Promise<string>` in this case. That the two platforms return different types for the same conceptual action is another topic entirely. 😛